compaction/compaction_strategy.cc

```diff
@@ -136,7 +136,7 @@ void size_tiered_backlog_tracker::refresh_sstables_backlog_contribution() {
     const auto& newest_sst = std::ranges::max(_all, std::less<generation_type>(), std::mem_fn(&sstable::generation));
     auto threshold = newest_sst->get_schema()->min_compaction_threshold();
 
-    for (auto& bucket : size_tiered_compaction_strategy::get_buckets(boost::copy_range<std::vector<shared_sstable>>(_all), _stcs_options)) {
+    for (auto& bucket : size_tiered_compaction_strategy::get_buckets(_all, _stcs_options)) {
```

Comment:
I think it's a net negative. Copying to a vector means the called function is not duplicated, and has simple iterators to work with.

Reply:
Especially as _all can be a quite complicated type - compound_sstable_set of partitioned_sstable_set.

Reply:
okay, will drop this in next version

Reply:
dropped in v6